inst/include/Rcpp/Reference.h

```diff
@@ -59,15 +59,15 @@ namespace Rcpp{
         Reference_Impl( const std::string& klass ) {
             SEXP newSym = Rf_install("new");
             Shield<SEXP> call( Rf_lang2( newSym, Rf_mkString( klass.c_str() ) ) );
-            Storage::set__( Rcpp_eval( call ) ) ;
+            Storage::set__( Rcpp_eval( call , Rcpp::internal::get_Rcpp_namespace()) );
```

Comment:
I guess the intention of this is to ensure that `new` is picked up from the `methods` namespace (which `Rcpp` exports)?

I would prefer something more explicit here, but I guess that's fine.